src/stores/WalletStore.js

```diff
@@ -155,6 +155,8 @@ export default class {
       runInAction(() => {
         this.txFees = txFees;
         this.txConfirmDialogOpen = true;
+        this.txToken = this.selectedToken;
+        this.txAmount = Number(this.withdrawAmount);
```

Comment:
remove these two, they are basically duplicates of `selectedToken` and `withdrawAmount`.

Reply:
Grr! You're so right! Why did I duplicate those two! [facepalm]